der/src/asn1/sequence.rs

```diff
@@ -28,12 +28,20 @@ impl<'a, T> Sequence<'a> for Box<T> where T: Sequence<'a> {}
 /// DER-encoded `SEQUENCE`.
 ///
 /// This is a zero-copy reference type which borrows from the input data.
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
 pub struct SequenceRef<'a> {
     /// Body of the `SEQUENCE`.
     body: &'a BytesRef,
 }
 
 impl<'a> SequenceRef<'a> {
+    /// Create a new [`SequenceRef`] from the provided DER bytes.
```

Comment:
I'm missing something, why can't we rely on the `DecodeValue::decode_value` instead?